### ISSUE TITLE
v0.8.0: node_modules in the server-function directory does not include all dependencies

### ISSUE THREAD

**Issue by fzia** [serverless-stack/open-next#75]
Using Monorepo nextjs and SST. The server-function directory inside the .open-next folder does not contain the required node_modules. This results in a number of dependency issues when run on AWS Lambda. Reverting to 0.7.0 fixes the issue.

**Comment by jayair** [comment]
Hmm that's weird. Can you post this on #help on Discord?

**Comment by roman-myroshnyk** [comment]
Had the same issue with yarn workspaces and SST, downgrading to 0.7.0 fixes it. 
```
  // Create a Next.js site
  const site = new NextjsSite(stack, "App", {
    path: "frontend",
    buildCommand: "npx open-next@0.7.0 build"
  });

```

**Comment by fwang** [comment]
@fzia @roman-myroshnyk This should be fixed in v0.9.0!

Let me know if it works for you.

**Comment by roman-myroshnyk** [comment]
@fwang Thank you for fast action.
Can confirm that this issue is fixed!
